.travis.yml

```diff
@@ -57,7 +57,6 @@ install:
     - sh autogen.sh
     - ./configure --enable-debug --with-zfs-headers=$PWD/../cstor/include --with-spl-headers=$PWD/../cstor/lib/libspl/include
     - make -j4
-    - make -f ../cstor/Makefile cstyle CSTORDIR=$PWD/../cstor
```

Comment:
`Makefile` in cstor directory gets created after running autogen.sh and configure commands in cstor directory.
So, this cstyle check can be moved down.
(or)
Below steps can be COPIED after the cstor branch is checked out few steps above
```
sh autogen.sh
./configure --with-config=user --enable-code-coverage=yes --enable-debug --enable-uzfs=yes --with-jemalloc --with-fio=$PWD/../fio --with-libcstor=$PWD/../libcstor/include
```
Above steps need to be done after checking out cstor repo

Reply:
oh. just did the same!

Reply:
right.. :) exactly at the same time..
I added a comment, and I got a 'Refresh' link for PR